### PR TITLE
Update link

### PR DESCRIPTION
**Pull request by samsycamore**

### CODE REVIEW

**Comment by thinkingserious** [comment]
Hello @samsycamore,<br /><br/>Thank you for the PR!<br />                                              <br />We will be reviewing this PR soon, in the mean                                              time, if you have not, please be sure to                                              <a href="https://github.com/sendgrid/docs#clas-and-cclas">                                             sign our CLA</a> so that we can merge your changes. Thanks again!                                             <br /><br />Team DX

**Comment by samsycamore** [comment]
Feel free to just change the link in the .md without merge

File: source/Integrate/Partners/magento.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ navigation:
   show: true
 ---
 
-This extension allows you to configure Magento 2.0 to send all emails using SendGrid. To get started, [click here](https://www.magentocommerce.com/magento-connect/sendgrid-email-delivery-1.html)!
+This extension allows you to configure Magento 2.0 to send all emails using SendGrid. To get started, [click here](https://marketplace.magento.com/sendgrid-email-delivery-simplified.html)!
```
Comment (mbernier) [inline source/Integrate/Partners/magento.md]: This looks good to me.

**Comment by thinkingserious** [comment]
Hello @samsycamore,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**